cc_proxy.go

```diff
@@ -98,15 +98,6 @@ func (p *ccProxy) register(pod Pod) ([]IOStream, error) {
 		return []IOStream{}, err
 	}
 
-	for i := 0; i < len(pod.containers); i++ {
-		ioStream, err := p.allocateIOStream()
-		if err != nil {
-			return []IOStream{}, err
-		}
-
-		ioStreams = append(ioStreams, ioStream)
-	}
-
```

Comment:
> Well the reason is that we are no longer calling register from StartPod, but from StartAgent instead. Several calls to connect will provide the tokens needed

Right, but I don't see those tokens/IOstream returned anywhere. They will be used as shim parameters, so they need to be passed down to the caller. Am I missing something again ?